kotlin/kotlin-quiz.md

```diff
@@ -39,7 +39,7 @@ class Airplane(private val name: String) : Aircraft() {
 #### Q4. Inside an extension function, what is the name of the variable that corresponds to the receiver object
 
 - [ ] The variable is named **it**
-- [ ] The variable is named **this**
+- [x] The variable is named **this**
```

Comment:
@adminazhar can you please share the reference link inside the *md file, thank you in advance! 

```suggestion
- [x] The variable is named **this**
[reference](https://www.programiz.com/kotlin-programming/extension-functions#:~:text=The%20class%20name%20is%20the,function%20refers%20the%20receiver%20object)
```